src/app/features/tech-record/components/tech-record-amend-vin/tech-record-amend-vin.component.ts

```diff
@@ -11,16 +11,16 @@ import { Store } from '@ngrx/store';
 import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
 import { updateVin, updateVinSuccess } from '@store/technical-records';
 import { TechnicalRecordServiceState } from '@store/technical-records/reducers/technical-record-service.reducer';
-import { take } from 'rxjs';
+import { Subject, take, takeUntil } from 'rxjs';
 
 @Component({
   selector: 'app-change-amend-vin',
-  templateUrl: './tech-record-amend-vin.component.html'
+  templateUrl: './tech-record-amend-vin.component.html',
+  styleUrls: ['./tech-record-amend-vin.component.scss']
```

Comment:
```suggestion
  templateUrl: './tech-record-amend-vin.component.html'
```
And delete the style sheet